Fatal Exception with <input type="file"/>

versions tested on: beta.3 and beta.6

When choosing a file via `<input type="file"/>` it works perfectly fine, but if you cancel (via Android's back button for example) the app crashes with the following exception:

```
D/Capacitor: Unable to find a Capacitor plugin to handle requestCode, trying Cordova plugins 9007
D/AndroidRuntime: Shutting down VM
E/AndroidRuntime: FATAL EXCEPTION: main
                  Process: com.myorg.myapp, PID: 26813
                  java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=9007, result=0, data=null} to activity {com.myorg.myapp/com.myorg.myapp.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.ClipData android.content.Intent.getClipData()' on a null object reference
                      at android.app.ActivityThread.deliverResults(ActivityThread.java:4166)
                      at android.app.ActivityThread.handleSendResult(ActivityThread.java:4209)
                      at android.app.ActivityThread.-wrap20(ActivityThread.java)
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1571)
                      at android.os.Handler.dispatchMessage(Handler.java:102)
                      at android.os.Looper.loop(Looper.java:241)
                      at android.app.ActivityThread.main(ActivityThread.java:6217)
                      at java.lang.reflect.Method.invoke(Native Method)
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
                   Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.ClipData android.content.Intent.getClipData()' on a null object reference
                      at com.getcapacitor.BridgeWebChromeClient$4.onActivityResult(BridgeWebChromeClient.java:152)
                      at org.apache.cordova.CordovaInterfaceImpl.onActivityResult(CordovaInterfaceImpl.java:159)
                      at com.getcapacitor.Bridge.onActivityResult(Bridge.java:689)
                      at com.getcapacitor.BridgeActivity.onActivityResult(BridgeActivity.java:189)
                      at android.app.Activity.dispatchActivityResult(Activity.java:6958)
                      at android.app.ActivityThread.deliverResults(ActivityThread.java:4162)
                      at android.app.ActivityThread.handleSendResult(ActivityThread.java:4209) 
                      at android.app.ActivityThread.-wrap20(ActivityThread.java) 
                      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1571) 
                      at android.os.Handler.dispatchMessage(Handler.java:102) 
                      at android.os.Looper.loop(Looper.java:241) 
                      at android.app.ActivityThread.main(ActivityThread.java:6217) 
                      at java.lang.reflect.Method.invoke(Native Method) 
                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865) 
                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755) 
```

--- a/android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java
+++ b/android/capacitor/src/main/java/com/getcapacitor/BridgeWebChromeClient.java
@@ -1,5 +1,6 @@
 package com.getcapacitor;
 
+import android.app.Activity;
 import android.content.ActivityNotFoundException;
 import android.content.Intent;
 import android.net.Uri;
@@ -149,7 +150,7 @@ public class BridgeWebChromeClient extends WebChromeClient {
         @Override
         public void onActivityResult(int requestCode, int resultCode, Intent intent) {
           Uri[] result;
-          if (intent.getClipData() != null && intent.getClipData().getItemCount() > 1) {
+          if (resultCode == Activity.RESULT_OK && intent.getClipData() != null && intent.getClipData().getItemCount() > 1) {
             final int numFiles = intent.getClipData().getItemCount();
             result = new Uri[numFiles];
             for (int i = 0; i < numFiles; i++) {

